docs/key_consumer_setup.md

```diff
@@ -24,6 +24,15 @@ Configure Salesforce to authenticate its HYOK requests to `distributey` using JW
        - `Unique Name`: this is the `KID` of the `JWT` token, thus must be unique. Recommended naming scheme: `jwt_kid_salesforce_serviceX`
      - Download the certificate and save it by its `Unique Name`. It must later be configured in `distributey` (`config/auth/`).
      - Salesforce provides a certificate from which `distributey` only needs its public key. Use this command to extract it: `openssl x509 -pubkey -noout -in jwt_kid_salesforce_serviceX.crt > jwt_kid_salesforce_serviceX.pub`
+     - Store the same public `jwt_kid_salesforce_serviceX.pub` key on the Vault JWT auth backend as part of `auth_jwt_validation_pubkeys` in the Terraform variables ([`./docker/terraform/terraform.tfvars`](../docker/terraform/terraform.tfvars)). For a production setup, this configuration needs to be done on the Vault server side, not on the development Vault in the Docker container:
```

Comment:
Can you mention that this is done (automatically) by `dev/write_jwt_to_tfvars.py` which is called by `dev/dev_setup.sh`?

I appreciate the callout about the production setup!